catalog-info.yaml

```diff
@@ -20,7 +20,8 @@ spec:
       name: eck-diagnostics
     spec:
       repository: elastic/eck-diagnostics
-      pipeline_file: ".buildkite/pipeline.yml"
+      provider_settings:
+        build_tags: true
```

Comment:
It is set to false by default. I hesitated to explicitly recall the other values for which the default values suit us well:
```yaml
    provider_settings:
        trigger_mode: "code"
        build_pull_requests: true
        build_branches: true
        build_tags: true
```